src/resources/resources.json

```diff
@@ -1853,6 +1849,20 @@
       "de": "Aktualisiert"
     }
   },
+  "recentContributions": {
+    "pageSubtitle": {
+      "en": "Open contributions made in the last 3 months.",
+      "de": "Open contributions made in the last 3 months."
+    },
+    "documentsTabTitle": {
+      "en": "Recent documents",
+      "de": "Neueste Dokumente"
+    },
+    "mediaLibraryTabTitle": {
+      "en": "Recent media",
+      "de": "Neueste Medien"
```

Comment:
I introduced the "Recent"/"Neueste" in there to make sure the title isn't overlooked by people working with the Maintenance page alot, so that they get them and their purposes confused. I'm not sure about the media tab title though, as on Maintenance we call it "Mediathek".